lib/Elastica/Util.php

```diff
@@ -197,6 +197,6 @@ public static function convertRequestToCurlCommand(Request $request)
      */
     public static function debugEnabled()
     {
-        return defined('DEBUG') && DEBUG;
+        return defined('ELASTICA_DEBUG') && ELASTICA_DEBUG;
```

Comment:
> They should work

I tried, but they don't.

Then I would like to remove this constant. One who want to get headers in response could pass `CURLINFO_HEADER_OUT` via curl options.